docs/manual/docs/administrator-guide/managing-users-and-groups/index.md

```diff
@@ -76,3 +76,124 @@ Rights associated with the roles are illustrated in detail in the list below:
     The Registered User has more access privileges than non-authenticated Guest users:
 
     -   Right to download protected data.
+
+## Role and feature Matrix
+
+The tables below show a comprehensive overview of Roles and Features, it explains in detail what role can do what in GeoNetwork.
+
+| Code  | Description |
+| ----- | ----- |
+| UI    | Feature visible in the UI, but not usable by the user profile |
+| EDIT  | The user can edit the metadata imported by the user |
+| DEL   | The user can delete the metadata imported by the user | 
+
+=== "Search"
+
+    |                                             | Anonymous user          |     Registered User     | Editor| Reviewer|       User Admin        |     Admini- strator     |
```

Comment:
Q: This did not look correct? "Admini- strator" --> "Administrator"

Reply:
Never mind I see the challenge 
<img width="783" alt="image" src="https://github.com/geonetwork/core-geonetwork/assets/629681/2ae664b1-9079-49a0-8f20-6479a2cb6a06">